content/departments/product-engineering/engineering/cloud/devops/managed/index.md

```diff
@@ -118,17 +117,3 @@ All customer credentials, secrets, site configuration, app and user configuratio
 Yes, you may disable the builtin authentication provider and only allow creation of accounts from configured SSO providers.
 
 However, in order to preserve site admin access for Sourcegraph operators, we need to add [Sourcegraph's internal Okta](./oidc_site_admin.md) as an authentication provider. Plesae reach out to our team prior disabling the builtin provider.
-
-### FAQ: "googleapi: Error 400: The network_endpoint_group resource ... is already being used"
-
-If `terraform apply` is giving you:
-
-```
-Error: Error when reading or editing NetworkEndpointGroup: googleapi: Error 400: The network_endpoint_group resource 'projects/sourcegraph-managed-$COMPANY/zones/us-central1-f/networkEndpointGroups/default-neg' is already being used by 'projects/sourcegraph-managed-$COMPANY/global/backendServices/default-backend-service', resourceInUseByAnotherResource
-```
-
-Or similar—this indicates a bug in Terraform where GCP requires an associated resource to be deleted first and Terraform is trying to delete (or create) that resource in the wrong order.
-
-To workaround the issue, locate the resource in GCP yourself and delete it manually and then `terraform apply` again.
-
-[#cloud-devops]: https://sourcegraph.slack.com/archives/C02KX975BDG
```

Comment:
the homepage for MI should be mostly external facing.

this is operation detail that only we care, so I moved it to the operation guide